### DISCUSSION TITLE
Doorbell

### DISCUSSION THREAD

**Discussion by Thorium9O** [Motion-Project/motion#1695] [General]
Has anyone set up a door bell with Motion? Looking for advice on how it works and how to set it up.
Thanks,

**Comment by leoheck** [comment]
I do. I created this project for myself. https://github.com/leoheck/peepcam
I am currently switching to Motion 4.5.1 + Raspbian Bullseye + Raspi Camera v1.3 (using libcamera). I am also using Telegram to get notifications on my smartphone. Once everything is running fine, I will update my changes to the peepcam repo.

**Comment by PanderMusubi** [comment]
May I create an issue that the code to send notification and image/video to Telegram is included/supported in Motion? See also https://github.com/leoheck/peepcam/blob/main/code/bin/send-to-telegram.py and https://github.com/Motion-Project/motion/discussions/1512
